notebook/services/api/api.yaml

```diff
@@ -814,6 +814,9 @@ definitions:
         type: string
         description: Last modified timestamp
         format: dateTime
+      size:
+        type: integer
+        description: "The size of the file or notebook in bytes. If no size is provided, defaults to None."
```

Comment:
Since this is describing a JSON API, it's probably better to refer to *null*, which is the JSON/Javascript name, rather than *None*, which is from Python. This isn't worth holding the PR up for, so I'll change it myself and then merge.